gulpfile.js

```diff
@@ -117,12 +117,7 @@ function genproto_group3_commonjs_strict(cb) {
 }
 
 
-function getClosureCompilerCommand(exportsFile, outputFile, keepSymbols) {
-  let compilationLevel = 'ADVANCED';
-  if (keepSymbols === true) {
-    compilationLevel = 'SIMPLE';
```

Comment:
what is the default level that we are no using, could we be explicit here?

maybe just leave a comment

Reply:
Added a comment. It's SIMPLE_OPTIMIZATIONS. There's some ambiguity in the docs over SIMPLE vs SIMPLE_OPTIMIZATIONS so I figure it was better to just revert to something similar to our initial state (nothing specified) rather than fiddle with it.